R/get_pvalue_position.R

```diff
@@ -159,7 +159,7 @@ get_y_position_core <- function(data, formula, fun = "max", ref.group = NULL, co
       length.out = ncomparisons
     )
   }
-  if(!is.null(y.trans)) y.position <- y.trans(y.position)
+  if(!is.null(y.trans)) y.position <- get(y.trans)(y.position)
```

Comment:
I think, this will not work if `y.trans` is specified as function instead of character. To add support for character, I think we need an  additional if block like this: 

```r
if(!is.null(y.trans)) {
  if(is.function(y.trans)) y.position <- y.trans(y.position)
  else if(is.character(y.trans)) y.position <- get(y.trans)(y.position)
}
```

Reply:
That's a good idea! Would you like me to submit a new PR?

Cheers, Rico

BTW: love the sthda site!